src/txn/txn_rollback_to_stable.c

```diff
@@ -1526,6 +1529,17 @@ __rollback_to_stable_hs_final_pass(WT_SESSION_IMPL *session, wt_timestamp_t roll
 
     WT_TRET(__wt_session_release_dhandle(session));
 
+    /*
+     * Truncate history store entries from the partial backup remove list. The list holds all of the
+     * btree ids that do not exist as part of the database anymore due to performing a selective
+     * backup.
```

Comment:
I suggest changing `performing a selective backup` to `performing a selective restore from backup.`